test-project/Cargo.toml

```diff
@@ -0,0 +1 @@
+1 

```

Comment:
`rust-buffer-project` uses `cargo locate-project`. The test for it doesn’t need anything beyond the presence of Cargo.toml in the directory, so I didn’t put anything in it. If that’s misleading I can put in some sample content.